kitten.js

```diff
@@ -1,47 +1,44 @@
 window.onload = function () {
-  var Vote = function() {
-    this.initialize = function() {
-      var $option1 = new ChooseCat();
-      var $option2 = new ChooseCat();
-      $option1.prependTo('#cat-pic');
-      if ($option1 === $option2) {
-        $option2 = new ChooseCat();
-        console.log($option2);
-      } else {
-      $option2.prependTo('#cat-pic2');
-      }
-  }
-  var game = new Vote;
-  Vote.initialize
-
-var catPicsArray = []; //loops through and selects each image
-  for (var i = 1; i <= 14; i++) {
-   catPicsArray.push(i+'.jpg');
-  };
-
-var ChooseCat = function () {
-  return $('<img src="images/' + catPicsArray[Math.floor(Math.random() * catPicsArray.length)] + '">')
-}
-
-
-
-
-//for tracking vote totals
-$('section.grid_6').on('click', function(){
-    var voteTotal = 0;
-    voteTotal+=
-    $('.total').append(voteTotal);
-  })
+  var pics;
+  var  Game = function() {
 
+    $.ajax({
+      url: 'https://api.imgur.com/3/album/CTPOV.json',
+      beforeSend: function(xhr) {
+        xhr.setRequestHeader('Authorization', 'Client-ID 11c372c0f3dfff2');
+      }
+    })
 
+    .done( function(pics) {
+      var catPicsArray = []; //loops through and selects each image
+      for (var i = 0; i < 14; i++) {
+        catPicsArray.push(pics.data.images[i].link);
```

Comment:
Good! but you might want to var catPicsArray outside of the AJAX code so the rest of the code can work with it.